src/main/java/com/artipie/asto/rx/RxStorage.java

```diff
@@ -97,10 +97,14 @@
     Completable delete(Key key);
 
     /**
-     * Start a transaction with specified keys.
+     * Runs operation exclusively for specified key.
      *
-     * @param keys The keys regarding which transaction is atomic
-     * @return Transaction
+     * @param key Key which is scope of operation.
+     * @param operation Operation to be performed exclusively.
+     * @return Completion or error signal.
      */
-    Single<RxTransaction> transaction(List<Key> keys);
+    Completable exclusively(
```

Comment:
@olegmoz if we agree to use `<T>` as `CompletionStage` paramter in `Storage` interface, then we could use `Single<T>` or `Maybe<T>` here